client/my-sites/checkout/checkout/style.scss

```diff
@@ -561,6 +561,10 @@
 	}
 }
 
+.cart__header .gridicons-cart {
+	transform: translate(.5px, .5px);
```

Comment:
Do you think we need `offset-adjust` here? It looks fine to me without it.

Reply:
Oh, I'm not sure...I was following the Gridicons guide in devdocs that said if you use icons at 18px they need offset adjust.

Reply:
Ohhh, I missed that. :D